guides/expression_language.md

```diff
@@ -0,0 +1,167 @@
+# Trento Checks Expression Language Reference
+
+A small, fast, easy-to-use scripting language and evaluation engine
```

Comment:
Should we put some reference to the rhai book?
It might help the users to find advanced use cases
https://rhai.rs/book/language/

Reply:
Added here adb85e15eb5d24e6269b37950fc799c779fc125a